Add introduction to workflows (#8)
inngest/docs: docs/workflows/workflows.md

---
sidebar_position: 1
---

# Workflows

<div class="tldr">
	<p><b>Workflows run your serverless code within Inngest.</b>  They can be set up to <b>run on a schedule</b>, or <b>run every time specific events are received</b>.</p>
	<p>A workflow consists of a series of serverless functions ("actions") that are called each time the workflows run.  Actions can be your own code or you can use our library of builtins.</p>
</div>

Workflows power your custom logic within Inngest.  They unlock the power of automating anything you need, either in response to events or on a schedule.

### What is a workflow?

A workflow is a series of actions which are triggered each time the workflow runs.  Workflows can be triggered via two methods:

> 1. In response to events ("event workflows"):  each time an event is received, a workflow is ran

Reviewer: ```suggestion
1. In response to events ("event workflows"):  each time an event is received, the workflow runs
```

2. On a specific schedule ("scheduled workflows"):  for example, every 15 minutes, or every day at 3AM.

> Importantly, **workflows are defined as code**.  You can write code to define a workflow, or **you can use our low code UI to write and customize your workflow**.

Reviewer: ```suggestion
Importantly, **workflows can be defined as code**.  You can write code to define a workflow, or **you can use our low code UI to write and customize your workflow**.
```

Author: Hmm, I think that while technically correct workflows _are actually_ defined as code, and the first is more precise. What are your thoughts around the change?
